src/components/DateTimePicker/Internal/ocpicker.module.scss

```diff
@@ -166,21 +181,20 @@
       height: $picker-cell-height-m;
       margin: 0;
       padding: 0;
-      font-family: $octuple-font-family;
+      font-family: var(--font-stack-full);
       font-size: $picker-font-size-m;
       font-weight: $text-font-weight-regular;
       line-height: $picker-cell-height-m;
       border-radius: 50%;
       outline: none;
       cursor: pointer;
       text-align: center;
-      transition: background $motion-duration-extra-fast,
-        border $motion-duration-extra-fast;
       width: $picker-cell-width-m;
 
       &:hover {
-        background: $picker-cell-background-color-hover;
-        color: $picker-cell-foreground-color-hover;
+        background: var(--picker-cell-background-color-hover);
+        color: var(--picker-cell-foreground-color-hover);
+        outline: 1px solid var(--picker-cell-selected-border-color-active);
```

Comment:
why the active color here? might as well make a hover color for this seemingly.

Reply:
Will look into this, figured as these are the same, may as well reuse the variable.

Reply:
Done